nucleus/common/common-util/src/main/java/com/sun/enterprise/universal/glassfish/ASenvPropertyReader.java

```diff
@@ -322,7 +323,7 @@ static private String getJavaRoot(String fileValue) {
                 return javaRootName; // we are already done!
 
             // try JAVA_HOME
-            javaRootName = System.getenv("JAVA_HOME");
+            javaRootName = System.getProperty("java.home");
```

Comment:
That's a real change here. You are switching from using ENV JAVA_HOME=... to java -Djava.home=...
Was that really a spotbugs warning?

Reply:
Ok. thats fine and I learned something new. Thank you.